src/telemetry.ts

```diff
@@ -29,6 +46,14 @@ export function createTrackingEvent(name: string, properties = {}): TelemetryEve
   }
 }
 
+export function sendCommandContentToTelemetry(commandId: string, message?: string): void {
```

Comment:
I think we need consistency in function names: `telemetryError ` and `sendCommandContentToTelemetry `